code/ui/blocks/src/blocks/Primary.stories.tsx

```diff
@@ -1,13 +1,16 @@
 import type { Meta, StoryObj } from '@storybook/react';
 import { Primary } from './Primary';
+import * as DefaultButtonStories from '../examples/Button.stories';
+import * as StoriesParametersStories from '../examples/StoriesParameters.stories';
 
-const meta = {
+const meta: Meta<typeof Primary> = {
```

Comment:
Curious why you changed from the `satisfies` operator to direct assignment of the type?

Reply:
Hi, thanks for reviewing my code! I changed it based off other PRs and off of Description.stories.tsx. But this is a good question, and after some research, I'll change it back to satisfies since it's just better for validation and type safety.